src/openms/source/ANALYSIS/ID/ConsensusMapMergerAlgorithm.cpp

```diff
@@ -32,11 +32,10 @@
 // $Authors: Julianus Pfeuffer $
 // --------------------------------------------------------------------------
 
+#include <include/OpenMS/ANALYSIS/ID/ConsensusMapMergerAlgorithm.h>
```

Comment:
`include/OpenMS` ?

Reply:
hmm good question. just moved it without thinking.